src/components/GuestList/GuestList.tsx

```diff
@@ -107,6 +100,8 @@ const GuestList: React.FC<WeddingData> = ({
     setGuests(filteredGuestList)
   }
 
+  useMemo(() => setGuests(guestList), [guestList])
```

Comment:
I'll have to learn more about how this hook works!

Reply:
I'm a little unclear of how it works exactly too, but I can definitely attempt to explain, and we can investigate it a bit more